### DISCUSSION TITLE
Tried an alternate op-amp, OPA2134, with success

### DISCUSSION THREAD

**Discussion by seanboyce** [ozel/DIY_particle_detector#53] [General]
The TLE2072 is a bit of a pain to find in my country. I ordered 2 from overseas, but am trying other op-amps as well.

The OPA2134 worked OK (I expect the OPA2132 would be fine too). The low noise figure is nice. It's a common op-amp for high quality audio that I can buy 'on the street' instead of importing parts. 

Reporting it here in case it's useful to others.

**Comment by ozel** [comment]
Hi Sean, good point about opamp availability and thanks for sharing! I tested a few back then and OPA2134 was one of the good candidates! Can you also trigger with them reliably at about at -24 mV?
The TL072(ACP) is another cheap alternative I am suggesting in the BOMs and often used together with the BPW34. About a third of the costs compared to the TLE2072CP but at doubled noise floor.  

Here are my notes from a small survey, stating usable trigger levels (falling edge) as observed on a decent Tek scope for a couple of minutes with a source:


TLE2072CP up +/- 50 mVPP noise with full battery, noise-free trigger of particle pulses at -37 mV possible
note there's also the TLE2072**A**CP version with reduced input voltage offset. However, the typical Vio value at root temperature is only slightly better than the one of TLE2072CP (0.9 vs. 0.65 mV). Prices are almost the same, both versions are a good option. Often one of the two version is easier available than the other.

TL072ACP: trigger at -80mV
TL072BCP: trigger at -90mV

MC33078PG (60 cent) -100 mV trigger or worse
OP275GPZ (2,80 EUR) -100 mV trigger or worse
opa2277pa (4,76 EUR) noise much slower but still rather high amplitude, trigger at -80 mV (GBW 1 MHz)
lt1112CN8#PBF (4.10 EUR) much less noise, trigger at -16 mV. noise is very slow due to low GBW (<1 MHz!)
OPA2134PA (4,14 EUR) less noise, trigger at -24 mV
(all prices from ~2019 within EU)

Where are you based regarding parts availability? Here in EU the TLE2072CP is currently 20% cheaper at Mouser than the  OPA2134PA also in higher quantities. Unfortunately it looks like TI is phasing that 'P' version, in easy to solder PDIP package, out of production while the TLE2072s are not leaving us anytime soon.

**Reply by seanboyce** [reply]
Ah, I'm in Vietnam. Mouser orders usually ship out of Thailand, so I've got significant postage and import duties. Usually around 100-200$, so it's only practical for larger part orders.

The OPA2134 I can hop on my motorbike and buy in the industrial market for 5-6 USD in unit quantities.

The OPA2132 works too, by the way. That's what I used to use in photodiode radiation detectors years ago, it used a very similar photodiode to the alpha detector.

Anyway, overall we get a different ecosystem of parts here. If you're curious, this is typical: https://www.thegioiic.com/

Regarding the test data, unfortunately I was short on parts and had to make some changes to the circuit. They are not improvements -- notably they increase noise a bit (e.g. no 40M resistors available here for the preamp). I'll run a test when I can, and report back though.
